test/e2e/framework.go

```diff
@@ -679,6 +680,7 @@ func (data *TestData) mutateFlowAggregatorConfigMap(ipfixCollector string, faClu
 	flowAggregatorConf = strings.Replace(flowAggregatorConf, "#externalFlowCollectorAddr: \"\"", fmt.Sprintf("externalFlowCollectorAddr: \"%s\"", ipfixCollector), 1)
 	flowAggregatorConf = strings.Replace(flowAggregatorConf, "#activeFlowRecordTimeout: 60s", fmt.Sprintf("activeFlowRecordTimeout: %v", aggregatorActiveFlowRecordTimeout), 1)
 	flowAggregatorConf = strings.Replace(flowAggregatorConf, "#inactiveFlowRecordTimeout: 90s", fmt.Sprintf("inactiveFlowRecordTimeout: %v", aggregatorInactiveFlowRecordTimeout), 1)
+	flowAggregatorConf = strings.Replace(flowAggregatorConf, "#includePodLabels: false", fmt.Sprintf("includePodLabels: %v", includePodLabels), 1)
```

Comment:
For e2e test, currently we are changing the config to include pod labels during e2e test and didn't add new test for not including pod labels situation to avoid making test longer, we would like to hear more options from @srikartati, do you think we need to add the test for the not including pod labels situation?

Reply:
That is fine but I see that the "false" case was skipped for unit tests. Maybe it can be added thereto get coverage for it.
 @zyiou plans to add flow aggregator restart test in e2e tests. This scenario of not including Pod labels could be covered then. Please take note of that.

Reply:
done